iptables-wrappers.yaml

```diff
@@ -0,0 +1,36 @@
+package:
+  name: iptables-wrappers
+  version: 2
+  epoch: 0
+  description: Wrapper scripts for using iptables in containers
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - ip6tables # this pulls in iptables as well
```

Comment:
this block is build time deps. do you mean to put this as runtime deps?

Reply:
it's required for both. i wasn't sure if i should add it to runtime deps as well since you'd typically use it with the `iptables`/`ip6tables` packages anyways.

Reply:
do you want me to add it to the runtime deps?

Reply:
looking at how it's done for the cilium package, it seems to have been added to both buildtime and runtime deps:

- https://github.com/wolfi-dev/os/blob/main/cilium-1.15.yaml#L18-L21
- https://github.com/wolfi-dev/os/blob/main/cilium-1.15.yaml#L49

i'll add it to runtime deps then

Reply:
added